test/integration/cel.v2/trainjob_crd_test.go

```diff
@@ -0,0 +1,155 @@
+package cel_v2
```

Comment:
I think, we can add those integration tests as part of `/test/integration/trainjob_controller_test.go`, similar to JobSet: https://github.com/kubernetes-sigs/jobset/blob/main/test/integration/controller/jobset_controller_test.go#L49
WDYT @akshaychitneni @tenzen-y ?

Reply:
Ack. Makes sense to me. I moved the tests to controller test

Reply:
Ideally, we should decouple both marker validation tests and reconciler tests.
But, after we implemented the webhook validations, let us revisit the directory structure.